.appveyor.yml

```diff
@@ -1,8 +1,15 @@
 environment:
   matrix:
-  - TARGET: x86_64-pc-windows-msvc
-  - TARGET: i686-pc-windows-msvc
-  - TARGET: i686-pc-windows-gnu
+    - TARGET: x86_64-pc-windows-msvc
+    - TARGET: i686-pc-windows-msvc
+    - TARGET: i686-pc-windows-gnu
+
+matrix:
+  allow_failures:
```

Comment:
Does this mean the build will always succeed? Windows is not a supported target by this project so not a problem if this build is broken but would be nice to know, I think.